src/main/java/com/iridium/iridiumskyblock/utils/SkinUtils.java

```diff
@@ -0,0 +1,61 @@
+package com.iridium.iridiumskyblock.utils;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import java.util.Base64;
+import java.util.HashMap;
+import java.util.UUID;
+
+public class SkinUtils {
+
+    private static HashMap<UUID, String> cache = new HashMap<>();
+
+    public static String getHeadData(UUID uuid) {
+        return cache.get(uuid);
+    }
+
+    public static String queryHeadData(UUID uuid) {
+        try {
+            Gson g = new Gson();
```

Comment:
You know my opinion about such short names by now, I guess :D

Reply:
I'm not the author of that code

Reply:
But you can change it 🤔